drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/main/java/org/drools/workbench/screens/dtablexls/backend/server/conversion/GuidedDecisionTablePopulater.java

```diff
@@ -220,7 +220,8 @@ private void addIndirectSourceBuildersColumns() {
                                             rhsBuilders);
         final RuleModel rm = RuleModelDRLPersistenceImpl.getInstance().unmarshal(rule,
                                                                                  Collections.emptyList(),
-                                                                                 dmo);
+                                                                                 dmo,
+                                                                                 true);
```

Comment:
There is not such method in the public interface https://github.com/kiegroup/drools/blob/master/drools-workbench-models/drools-workbench-models-commons/src/main/java/org/drools/workbench/models/commons/backend/rule/RuleModelPersistence.java Is there any other PR related to this ensemble?